src/gluonts/transform/convert.py

```diff
@@ -679,7 +678,7 @@ def _fill(target: np.ndarray, expected_length: int) -> np.ndarray:
 
 
 def cdf_to_gaussian_forward_transform(
-    input_batch: DataEntry, outputs: Tensor
+    input_batch: DataEntry, outputs: np.ndarray
```

Comment:
@mbohlkeschneider could you check that these changes to the CDF transform are OK?

Reply:
Should be fine.